cpu_map_keyme2560.h

```diff
@@ -49,9 +49,9 @@
 
 /* Chip select for SRAM */
 #define SCS_SRAM_PORT           PORTC
-#define SCS_SRAM_PIN            PC3
+#define SCS_SRAM_PIN            PC2
 #define SCS_SRAM_DDR            DDRC
-#define SCS_SRAM_DDR_PIN        DDC3
+#define SCS_SRAM_DDR_PIN        DDC2
```

Comment:
Did these pins change between the old control board and the new one? If so, do we need to make these values depend on something external, so we can still compile stuff for both old and new control boards?